ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -279,6 +283,7 @@ test-suite test-storage
                        -Wmissing-export-lists
                        -Wno-unticked-promoted-constructors
                        -fno-ignore-asserts
+                       -threaded
```

Comment:
Do you know what did we add this?

Reply:
The LMDB bindings won't work if we don't set the `-threaded` option

```bash
$ cabal run ouroboros-consensus-test:test-storage -- -p "LMDB" 

TDBOpening
ouroboros-storage
  Storage
    LedgerDB
      HD
        LMDB
          simple insert and read: FAIL
            Exception: user error (RTS doesn't support multiple OS threads (use ghc -threaded when linking))
            Use -p '/LMDB/&&/simple insert and read/' to rerun this test only.
      OnDisk
        LedgerSimple-LMDB:        FAIL
          *** Failed! Exception: 'user error (RTS doesn't support multiple OS threads (use ghc -threaded when linking))' (after 1 test and 1 shrink):
          SecurityParam 0
          Commands { unCommands = [] }
          Use --quickcheck-replay=160168 to reproduce.
          Use -p '/LMDB/&&/LedgerSimple-LMDB/' to rerun this test only.

2 out of 2 tests failed (0.00s)
```